react-app/src/App.js

```diff
@@ -14,27 +14,19 @@ function App() {
   };
 
   const handleInputChange = (event) => {
-    if(event.target.name === "long"){
-      setLongUrl(event.target.value);
-    }else if(event.target.name === "short"){
-      setShortUrl(event.target.value);
-    }
+    setLongUrl(event.target.value);
   }
 
   const longToShort = async () => {
     console.log(longUrl);
     try {
       const response = await axios.post(`http://localhost:7000/longToShort?url=${longUrl}&&method=${selectedOption}`);
```

Comment:
The endpoint should be put in a config file instead of hard-code it inside the business logic

Reply:
End points are now stored in a seperate config file (configData.json), and is imported when used.